compiler/src/iree/compiler/Dialect/Flow/Transforms/FuseDequantizationMatmul.cpp

```diff
@@ -56,6 +56,20 @@ static LogicalResult fuseDequantAndMatmul(RewriterBase &rewriter,
   if (failed(maybeFusedRegionOp))
     return failure();
 
+  if (fill && *fill) {
+    FailureOr<DispatchRegionOp> maybeFusedFillRegionOp;
+    if (fill.value()->hasOneUse()) {
+      maybeFusedRegionOp =
```

Comment:
Nit: In thoery you dont need to do a separate check for move vs clone. Clone always will basically make the fill dead, but in theorry you shouldnt need to do this anyway.

Reply:
I had the same comment before for the dequant and don't really have an opinion here. I'll go with whichever way you prefer.

Reply:
my suggestion is just clone (that has lesser chances of going awry, move has impact on use-def chains). If that works for you, then thats enough. If it doesnt, then add the move.

Reply:
Makes sense. I'll switch the dequant to clone also then.